ip.go

```diff
@@ -85,13 +91,16 @@ const (
 // IPAddressReservation is created when user sends IP reservation request for a project (considering it's within quota).
 type IPAddressReservation struct {
 	IpAddressCommon
-	Assignments []*IPAddressAssignment `json:"assignments"`
-	Facility    *Facility              `json:"facility,omitempty"`
-	Available   string                 `json:"available"`
-	Addon       bool                   `json:"addon"`
-	Bill        bool                   `json:"bill"`
-	State       IPReservationState     `json:"state"`
-	Description *string                `json:"details"`
+	Assignments  []*IPAddressAssignment `json:"assignments"`
+	Facility     *Facility              `json:"facility,omitempty"`
+	Available    string                 `json:"available"`
+	Addon        bool                   `json:"addon"`
+	Bill         bool                   `json:"bill"`
+	State        IPReservationState     `json:"state"`
+	Description  *string                `json:"details"`
+	Enabled      bool                   `json:"enabled"`
```

Comment:
I see that this field is returned in the portal. I don't know what this actually indicates.

Reply:
Enabled represents wether or not the IP is assigned to a Port. Ports could represent Devices, Gateways, or other resource types (if not now, then later). If Enabled is set, you generally won't be able to assign this IP Address elsewhere without unassigning it from some associated resource first.